Makefile

```diff
@@ -1,10 +1,10 @@
 TOPIC ?= gans
+TOPIC ?= cotracker
```

Comment:
here there is no need to add another default. I just added it in case someone runs the make command without setting a different TOPIC variable